server/tailsql/static/script.js

```diff
@@ -20,6 +21,15 @@
         }
     })
 
+    body.addEventListener("keydown", (evt) => {
+        if (evt.ctrlKey) {
```

Comment:
I think on Windows `ctrl + num` lets you go through to that number of tab (e.g. `ctrl 1` = switch to first tab), I'm not sure if this will either hijack that behaviour or switch the input source AND send you to the first tab.

Reply:
That's an interesting point, let me try it on Windows. 🙂

Reply:
As you predicted, it has the effect of doing both. 😆 

What I really wanted was "the key that macOS calls option and windows calls alt", but on macOS "option" gets co-opted for composition.

Reply:
You could prevent the tab switching with `e.preventDefault()` but I don't know if that would be annoying for people who use windows haha

Reply:
I switched it to Alt instead, once I realized I could work around the composition issue by checking codes instead of keys. @phirework how terrible is that to do?

Reply:
It's probably fine! Esp for an internal tool I wouldn't worry too much about it, let's see if we get bugreports 😅

Reply:
Interestingly, I tried using `preventDefault` before I saw your note, and it did _not_ actually prevent the OS from firing the app-level action. I tried a few other combinations of that and stopPropagating, switching to keydown, etc., but none of them affected it.

Reply:
Hijacking upon hijacking 😂